Readme.md

```diff
@@ -0,0 +1,6 @@
+The official android app for MusicBrainz. MusicBrainz for Android lets users search metadata all of the the metadata available on MusicBrainz. Users can scan barcode to search albums. Registered users can view, edit and modify their collections. Users can also tag their audio files.
+# What's next
+* The tagger is still in initial stages. The app needs to achieve complete usability with the desktop version of Picard. Many audio formats are not suppored yet. There is no support for Cover Art. In the supported formats, not all tags are currently be allowed. At many points, the JAudioTagger library being used underneath to interact with tags limits the usability. Either work has to be done at developing at a new library for JAVA which fulfills the needs or otherwise a suitable C/C++ can be used with the help of Andrid NDK and JNI.
```

Comment:
Maybe adding the last commit for GSoC is useful, as noted in the documentation:

> If the pull request is going to have more work done after GSoC is over, make sure the last GSoC commit is noted.

```suggestion
The last commit for Google Summer of Code 2019 was ac470a8 However there is more work to be done after GSoC, like:
* The tagger is still in initial stages. The app needs to achieve complete usability with the desktop version of Picard. Many audio formats are not suppored yet. There is no support for Cover Art. In the supported formats, not all tags are currently be allowed. At many points, the JAudioTagger library being used underneath to interact with tags limits the usability. Either work has to be done at developing at a new library for JAVA which fulfills the needs or otherwise a suitable C/C++ can be used with the help of Andrid NDK and JNI.
```